.github/workflows/linear-pr-links.yml

```diff
@@ -0,0 +1,104 @@
+name: Update PR Description with Linear Link
+
+on:
+  push:
+    branches:
+      - '*'  # Triggers on push to any branch that has an open PR
```

Comment:
I don't think this is the right trigger
- I think this will trigger on any push to any branch, whether or not it has a pr (like master or release branches)
- if you trigger on push, then open the PR, it won't ever be able to edit the description (since it doesn't exist yet)

Reply:
👍